pkgs/top-level/python-packages.nix

```diff
@@ -4256,6 +4256,10 @@ let
     };
   };
 
+  # py3k disabled, see https://travis-ci.org/NixOS/nixpkgs/builds/48759067
+  graph-tool = if isPy3k then null
```

Comment:
This looks odd. All other packages with this property add an `disabled = isPy3k` attribute into the expression itself.